core/src/main/java/nl/inl/util/Json.java

```diff
@@ -94,8 +96,9 @@ public static String getString(JsonNode parent, String name, String defVal) {
      * @return the boolean value
      */
     public static boolean getBoolean(JsonNode parent, String name, boolean defVal) {
-        if (parent.has(name))
-            return parent.get(name).booleanValue();
+        if (parent.has(name)) {
+            return parent.get(name).asBoolean(defVal);
```

Comment:
Do we need to relax these this much? Are non-boolean values being serialized where we would expect a boolean?

Reply:
From testing this, it seems like in some cases boolean values will get serialized in their canonical text form(this seesm to me more a library decision, could not find anywhere in the spec that was the case), that will create a `TextNode` that needs to be parsed, which I agree, is a sad side effect of the canonicalization.

These methods are protected with  default values that Blacklab uses as sanity check in many cases, diminishing the risk of bad data impacting the rest of blacklab. This, however, could potentially mask erroneous input, making harder to find bugs. :/